components/charity/modal/CharityBody.tsx

```diff
@@ -1,7 +1,12 @@
 import React from "react";
 import Image from "next/image";
+import { CharityEvent } from "data/types";
 
-const CharityBody = ({ event }) => {
+interface CharityBodyProps {
```

Comment:
Still think it'd be nice to do this - would avoid needing an interface in each file.

The interface is identical in CharityBody, CaseCompBody, and CaseCompBody2022, and it's never used with a type other than CharityEvent, so we don't even need a generic. Could just put the interface in types.ts, maybe called `CharityEventWrapper` or similar. Or, could export it from e.g. CharityBody.tsx (not as keen on this because it's less tied to one particular file).

CharityCard is also very similar, would just become a [type intersection](https://www.typescriptlang.org/docs/handbook/2/objects.html#intersection-types) or extending both that proposed charity event type and ClickableEvent.

Reply:
just saw this - putting it in